src/main/java/net/dv8tion/jda/api/utils/MarkdownSanitizer.java

```diff
@@ -71,7 +71,7 @@
     private static final int ESCAPED_QUOTE_BLOCK = Integer.MIN_VALUE | QUOTE_BLOCK;
 
     private static final Pattern codeLanguage = Pattern.compile("^\\w+\n.*", Pattern.MULTILINE | Pattern.DOTALL);
-    private static final Pattern quote = Pattern.compile("> +\\S.*", Pattern.DOTALL | Pattern.MULTILINE);
+    private static final Pattern quote = Pattern.compile("> +(\\S.*)*", Pattern.DOTALL | Pattern.MULTILINE);
```

Comment:
What's the point of this group? Just do `> +.*`

Reply:
No point, just my not so good experience with regex.